pkg/loader/load.go

```diff
@@ -300,6 +302,10 @@ func (s *loaderImpl) refreshTableInfo(schema string, table string) (info *tableI
 	return
 }
 
+func (s *loaderImpl) evitTableInfo(schema string, table string) {
```

Comment:
In `master`, we already have a method called `invalidateTableInfo` for invalidating the cached table info.

Reply:
[remove invalidateTableInfo
](https://github.com/pingcap/tidb-binlog/pull/892/commits/7aa677b6cce31bf0d5439b94e219c52dda5e5ed7)

Reply:
I suppose we should remove `evitTableInfo` instead, `evit` is not even an English word ...

Reply:
https://github.com/pingcap/tidb-binlog/pull/892/commits/686ef25a34a4c242fd8e29bd1ffb3ab89bcd93e4
use `evict`